firebase/build.boot

```diff
@@ -4,7 +4,8 @@
 
 (require '[cljsjs.boot-cljsjs.packaging :refer :all])
 
-(def +lib-version+ "5.0.4")
+(def +lib-version+ "5.4.2")
+'use strict';
```

Comment:
You have accidentally added a line of JS here.

Reply:
Oops, thanks.